client/src/main/scala/drt/client/components/MonthlyStaffing.scala

```diff
@@ -234,6 +241,15 @@ object MonthlyStaffing {
         ^.onClick ==> handleShiftEditForm
       ))
       <.div(
+        if (!props.isStaffShiftPage) {
+          if (props.showAddShifts) {
```

Comment:
Is this logic inverted? If `showAddShifts` is true then we're giving an emptydom